IES Specification Docs/Information Exchange Standard r4.2.0.n3

```diff
@@ -2362,7 +2269,7 @@ ies:Parked rdfs:comment "A temporal state of a Vehicle where it is not moving. E
 ies:Parked rdfs:subClassOf ies:VehicleState . 
 ies:ParticularPeriod rdf:type rdfs:Class . 
 ies:ParticularPeriod sparx:guid "{2173F463-524C-457c-B106-51322F64F122}" . 
-ies:ParticularPeriod rdfs:comment "A PeriodOfTime that is a specific, contiguous extent of time.IMPORTANT NOTE: The URI of a ParticularPeriod shall be encoded as follows:http://iso8601.iso.org/2007-01-18T15%3A00%3A00Where the content after the / is % encoded (e.g. encodeURIComponent in javascript). In the example above, the unencoded content would be 2007-01-18T15:00:00The reason behind using a URI is that receiving systems can resolve the periods of time and de-duplicate. Examples:Tuesday 28th August 20182016December 1944" . 
+ies:ParticularPeriod rdfs:comment "A PeriodOfTime that is a specific, contiguous extent of time. IMPORTANT NOTE: The URI of a ParticularPeriod shall be encoded in UTC (Coordinated Universal Time) and as follows:http://iso8601.iso.org/20070118T153000Where the content after the / is encoded without punctuation and without the trailing Z. In the example above, the punctuated equivalent would be 2007-01-18T15:30:00ZThe reason behind using a URI is that receiving systems can resolve the periods of time and de-duplicate. Examples:Tuesday 28th August 20182016December 1944" . 
```

Comment:
Line breaks are permitted in N3 literals if use """ syntax. Consider using these to make comments more readable. Linebreaks appear to be dropped in all rdfs:comment values. --Lisa

Reply:
See my comment below. This will be addressed for the final release of 4.3